src/language.cpp

```diff
@@ -301,4 +301,7 @@ void setTranslator(OUTPUT_LANGUAGE_t langName)
 #endif
 #endif
   }
+
+  QCString msg = theTranslator->updateNeededMessage();
+  if (!msg.isEmpty()) warn_uncond("%s", qPrint(msg));
```

Comment:
As is it is good that the user is informed about a not updated language, but doing this through the `warn...` mechanism is not good as when the user runs a CI job and has set `WARN_AS_ERROR` to `YES` or `FAIL_ON_WARNINGS` the job will stop.

It is probably netter to use the `msg` mechanism (though the user will not see the message when `QUIET=YES`).

Reply:
This change simply reinstates the previous behavior, i.e. versions before 1.9.3 *did* error out for the described settings (I presume).

If that is to be changed, should it be done here or in a separate MR?

Reply:
I don't know why @doxygen did take it out, but when we would reinstate it I think we should take into consideration the effect effect regarding the `warn` versus `msg` (I think the old behavior has some negative side effects, especially for languages that are nearly up to date). 
I think @doxygen should decide whether it should be reinstated and when reinstated in which form.

Reply:
I think this warning was accidentally removed. I also have a preference for using `msg` for the reason @albert-github  mentioned.

Reply:
Added a second commit that downgrades the severity.

Reply:
Whoops, now it compiles...

Reply:
I should have said "now it compiles for me". I can't see why the [miktexsetup_standalone](https://github.com/doxygen/doxygen/runs/5346726527?check_suite_focus=true#step:11:2) command in the [Windows MSVC Debug](https://github.com/doxygen/doxygen/runs/5346726527) pipeline failed. I think this is a sporadic failure, unrelated to the actual code change. I don't know how to trigger a rerun, though.

Reply:
As the problem is just at one platform don't worry about it it is indeed a sporadic failure.